README.md

```diff
@@ -1,45 +1,75 @@
-# Catapult-explorer
+# catapult-explorer
 
-## Setup Client
-```
-cd client
-npm install
-```
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-#### Compiles and hot-reloads for development
-```
-cd client
-npm run serve
-```
+The nem2 Command Line Interface (CLI) is a unified tool to interact with the NEM2 Blockchain.
 
-#### Compiles and minifies for production
-```
-cd client
-npm run build
-```
+This tool will enable you perform the most common and regular used actions in NEM2 (a.k.a Catapult)
```

Comment:
It would be nice to describe first a little bit the architecture of the package, clarifying the client and server responsibilities:

Example:

## Architecture

``name-of-library`` is composed of three independent parts.

### client

[describe package responsability]

### server

[describe package responsibility]